docs/sphinx/sections/api/apidocs/external-assets.rst

```diff
@@ -0,0 +1,85 @@
+External Assets (Experimental)
+==============================
+
+Instance API
+------------
+
+External asset events can be recorded using :py:func:`DagsterInstance.report_runless_asset_event` on :py:class:`DagsterInstance`.
+
+*Example*
+.. code-block:: python
+    from dagster import DagsterInstance, AssetMaterialization, AssetKey
```

Comment:
```suggestion
.. code-block:: python

    from dagster import DagsterInstance, AssetMaterialization, AssetKey
```

Sphinx is yelling about this (`Error in "code-block" directive: maximum 1 argument(s) allowed`) - tested locally; adding a blank line here should fix it

Reply:
Also, you'll need to add `sections/api/apidocs/external-assets` to `index.rst` to get this to build - just mentioning it since I don't see it on the PR yet and I had to do it to get it to build locally